src/main/java/org/jabref/logic/ai/summarization/SummariesService.java

```diff
@@ -59,6 +59,12 @@ public SummariesService(AiPreferences aiPreferences,
         this.taskExecutor = taskExecutor;
     }
 
+    /**
+     * Start generating summary of a {@link BibEntry}, if it was already generated.
```

Comment:
I think, there is something missing before "if it was"

```suggestion
     * Start generating summary of a {@link BibEntry}, independent of whether if it was already generated.
```

Reply:
Oops, it should mean: if it **wasn't**

Reply:
If it wasn't, it will start summarization task and put a `ProcessingInfo` in the map.

If it is generating, it will return current `ProcessingInfo`

Reply:
> Oops, it should mean: if it **wasn't**

Just reword and commit 😅